src/drawing/backend_impl/bitmap.rs

```diff
@@ -151,6 +153,7 @@ impl<'a> BitMapBackend<'a> {
         frame_delay: u32,
     ) -> Result<Self, BitMapBackendError> {
         Ok(Self {
+            _pix: PhantomData,
```

Comment:
And I believe this change doesn't support pixel format other than RGB888, unless we change the fast pixel manipulation code as well. See function `blend_rect_fast`, `fill_rect_fast`